libimagtodo/src/read.rs

```diff
@@ -0,0 +1,67 @@
+
+
+use libimagstore::storeid::StoreIdIterator;
+use libimagstore::store::{FileLockEntry, Store};
+use libimagstore::storeid::StoreId;
+use module_path::ModuleEntryPath;
+use error::{TodoError, TodoErrorKind};
+
+use std::result::Result as RResult;
+
+pub type Result<T> = RResult<T, TodoError>;
+
+pub struct Read<'a> {
+	 entry: FileLockEntry<'a>,
+}
+
+pub fn all_uuids(store: &Store) -> Result<ReadIterator> {
```

Comment:
ok, here's my comments:
Please rename this functions to "all_todos".
Also add a filter to iter, so only taskwarrior uuid's are given back, in pseudocode somewhat like this:
`ReadIterator::new(store, iter.filter(|id| is_in_subdirectory("tw", id)))`
Also, there's already the libimagtodo::task::Task type, which is a wrapper for an FileLockEntry. using that one might be more appropriate. Additionally, rename ReadIterator to TaskIterator.
A minor comment: if you're working with vim, please expand your tabs to 4 whitespaces and do a :retab!